SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

```diff
@@ -45,6 +58,20 @@ class SPATIALGDK_API USpatialDispatcher : public UObject
 	UPROPERTY()
 	USpatialStaticComponentView* StaticComponentView;
 
-	UPROPERTY()
-	TMap<uint32, UOpCallbackTemplate*> UserOpCallbacks; // key has to be explicit primitive or UHT complains
+	struct UserOpCallbackData
+	{
+		Worker_ComponentId ComponentId;
+		Worker_OpType OpType;
+		UserCallback<const Worker_Op*> Callback;
+	};
+
+	// This index is incremented and returned every time the AddOpCallback function is called.
+	// These indexes enable you to deregister callbacks using the RemoveOpCallback function. 
+	// RunUserCallbacks is called by the SpatialDispatcher and executes all user registered 
+	// callbacks for the matching component ID and network operation type.
+	CallbackId NextCallbackId;
+	CallbackId AddGenericOpCallback(Worker_ComponentId ComponentId, Worker_OpType OpType, UserCallback<const Worker_Op*> Callback);
```

Comment:
Can we order these so that structs are above functions, which are above fields.

Reply:
Do you mean this for everything in the `private:` scope or just everything related to the subject of user registered callbacks?

Reply:
Within modifier scope.